helm/values-dev.yaml

```diff
@@ -45,10 +45,10 @@ microservice-chart:
   resources:
     requests:
       memory: "512Mi"
-      cpu: "200m"
+      cpu: "300m"
     limits:
       memory: "512Mi"
-      cpu: "200m"
+      cpu: "500m"
```

Comment:
let's increase by only 100, let's try with 400

Reply:
Updated in 4185fd9